### PR TITLE
Added changelog for 1.13.0

### PR DESCRIPTION
**Pull request by pimjansen**
### What is the reason for this PR?

Release 1.13.0

- [ ] A new feature
- [ ] Fixed an issue (resolve #ID)

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)
- [x] New features and changes are [documented](https://github.com/FakerPHP/fakerphp.github.io)

### Summary of changes

Next minor update before the end of year.

### Review checklist

- [x] All checks have passed
- [ ] Changes are approved by maintainer

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+- Improve Austrian social security number generation (#88)
+- Move US phone numbers with extension to own method (#91)
+- Add UK National Insurance number generator (#89)
+- Fix en_SG phone number generator (#100)
+- Fix link to VAT documentation in nl_NL Company provider (#103)
+- Remove usage of mt_rand (#87)
+- Remove whitespace from beginning of el_GR phone numbers (#105)
+- Building numbers can not be 0, 00, 000 (#107)
+- Add 172.16/12 local IPv4 block (#121)
+- add JCB card (#124)
```
Comment (Nyholm) [inline CHANGELOG.md:29]:
```suggestion
- Add JCB credit card type (#124)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+- Feat/add at ssn (#79)
+- Fix typo in id_ID Address (#83)
+- Austrian post codes (#86)
+- Add internet provider slug tests (#85)
+- updated Polish data (#70)
+- Improve Austrian social security number generation (#88)
+- Move US phone numbers with extension to own method (#91)
+- Add UK National Insurance number generator (#89)
+- Fix en_SG phone number generator (#100)
+- Fix link to VAT documentation in nl_NL Company provider (#103)
```
Comment (Nyholm) [inline CHANGELOG.md:24]: This is just a small change in a dock block. 

```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+- Move AT payment provider logic to de_AT (#72)
+- Fix wiktionary links (#73)
+- Fix AT person links (#74)
+- Fix AT cities (#75)
+- Deprecate at_AT providers (#78)
+- Feat/add at ssn (#79)
+- Fix typo in id_ID Address (#83)
+- Austrian post codes (#86)
+- Add internet provider slug tests (#85)
+- updated Polish data (#70)
```
Comment (Nyholm) [inline CHANGELOG.md:19]:
```suggestion
- Updated Polish data (#70)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+- Fix greek phone numbers (#16)
+- Move AT payment provider logic to de_AT (#72)
+- Fix wiktionary links (#73)
+- Fix AT person links (#74)
+- Fix AT cities (#75)
+- Deprecate at_AT providers (#78)
+- Feat/add at ssn (#79)
+- Fix typo in id_ID Address (#83)
+- Austrian post codes (#86)
+- Add internet provider slug tests (#85)
```
Comment (Nyholm) [inline CHANGELOG.md:18]: This is just updates to the test. 
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+
+- Feature/pl pl license plate (#62)
+- Fix greek phone numbers (#16)
+- Move AT payment provider logic to de_AT (#72)
+- Fix wiktionary links (#73)
+- Fix AT person links (#74)
+- Fix AT cities (#75)
+- Deprecate at_AT providers (#78)
+- Feat/add at ssn (#79)
+- Fix typo in id_ID Address (#83)
```
Comment (Nyholm) [inline CHANGELOG.md:16]:
```suggestion
- Fix typos in id_ID Address (#83)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+on the codebase on both tests and consistency.
+
+- Feature/pl pl license plate (#62)
+- Fix greek phone numbers (#16)
+- Move AT payment provider logic to de_AT (#72)
+- Fix wiktionary links (#73)
+- Fix AT person links (#74)
+- Fix AT cities (#75)
+- Deprecate at_AT providers (#78)
+- Feat/add at ssn (#79)
```
Comment (Nyholm) [inline CHANGELOG.md:15]:
```suggestion
- Add at `ssn()` for Austrian `Person` (#79)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,35 @@
 # CHANGELOG
 
+## 2020-12-17, v1.13.0
```
Comment (Nyholm) [inline CHANGELOG.md:3]:
```suggestion
## 2020-12-18, v1.13.0
```

**Review by Nyholm — COMMENTED** [review]
Thank you. I've just got a few small things

File: CHANGELOG.md (hunk 6 above)
Comment (bramceulemans) [reply CHANGELOG.md:15]: Add at? I'd recommend
```suggestion
- Add Austrian `ssn()` to `Person` provider (#79)
```

**Comment by pimjansen** [comment]
> Thank you. I've just got a few small things

@Nyholm  Ah yeah i just copied the commit msg. Applied all the changes

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**